dist/svg.js

```diff
@@ -1701,12 +1701,12 @@ SVG.FX = SVG.invent({
      *  @return this
      */
   , step: function(ignoreTime){
+      var durDivSpd = this.situation.duration/this._speed
 
       // convert current time to position
-      if(!ignoreTime) this.pos = this.timeToPos(+new Date)
-      else {
-        // If the method at was called with a position above 1, we set it back to 1
-        if(this.pos > 1) this.pos = 1
+      if(!ignoreTime) {
+        this.pos = this.timeToPos(+new Date)
+      } else {
         if(this.situation.reversed) this.pos = 1 - this.pos
```

Comment:
I can spot the same statement at line 1753. It looks like this statement would un-done there